engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/details/views/repository-manage-docker-manifest-lists.html

```diff
@@ -16,17 +16,15 @@
 
       <thead>
         <tr bst-table-head>
-          <th bst-table-column><span translate>Digest</span></th>
           <th bst-table-column><span translate>Tags</span></th>
-          <th bst-table-column><span translate>Schema Version</span></th>
+          <th bst-table-column><span translate>Digest</span></th>
+          <th bst-table-column><span translate>Platforms</span></th>
```

Comment:
I can't search/filter the table on platforms.

"Field 'platform' not recognized for searching!"

It should probably work in a similar way to tags.

Reply:
This also applies to the docker manifests page.

Reply:
Will do in separate PR https://projects.theforeman.org/issues/24758